_posts/2019-07-19-mock-parse-buildlog.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Mock can now parse the log from build
+author: Silvie Chlupova
+layout: post
+---
+
+Into [mock](https://github.com/rpm-software-management/mock) has been added a new feature called [mock-parse-buildlog](https://github.com/rpm-software-management/mock/blob/devel/mock/py/mock-parse-buildlog.py).
+
+This piece of code will help you identify what caused your build to fail. Currently, mock-parse-buildlog knows only two situations that could occur and cause that a build fail. First is that some files are in %buildroot but not in %files and the second one is that some files are in %files but not in %buildroot.
+
+Part of the code has been taken from rebase-helper.
+
+#### An example of how it works
+
+Run as:
+
+`mock-parse-buildlog --path /path/to/log/build.log`
+
+And the result is:
+
+```Error type: Build failed because problematic files are in %buildroot but not in %files```
+
+```Problematic files:
```

Comment:
Since this is supposed to be a single piece of an example output, I would change those ``` and open them a line before "Error type:" and close them after "[...]". It will render much prettier.

(Alternatively, if you don't want to mess with ```, you can just indent the whole code block four spaces to the right).